pkg/controller/controller.go

```diff
@@ -187,6 +191,25 @@ func (c *Controller) ensureK8sResources(funcObj *kubelessApi.Function) error {
 	}
 	funcObj.ObjectMeta.Labels["function"] = funcObj.ObjectMeta.Name
 
+	controllerNamespace := os.Getenv("KUBELESS_INSTALLED_NAMESPACE")
+	cm, err := c.clientset.CoreV1().ConfigMaps(controllerNamespace).Get("kubeless-config", metav1.GetOptions{})
+	if err == nil {
```

Comment:
it's better to examine and log out every failure situations might happen rather than skipping. 

```
if err != nil {
   logrus.Errorf("Can't get configmap kubeless-config at namespace %s: %v", controllerNamespace, err)
   return err
}
```

Reply:
@ngtuna actually this is redundant. If the `ConfigMap` is not present or cannot be read by `kubeless-controller` then controller fails to start with `Error: configmaps "kubeless-config" not found` hence we don't even reach here. May be, I should replace `err` with `_`.